nsqd/nsqd.go

```diff
@@ -504,6 +545,13 @@ func (n *NSQD) Notify(v interface{}) {
 			n.Unlock()
 		}
 	})
+
+	n.waitGroup.Wrap(func() {
+		select {
+		case <-n.exitChan:
+		case n.gossipChan <- v:
```

Comment:
nah i'm ok; the larger the block of code, the less i like single letter variables (`v` in this case), but it's fine.